android/src/main/java/com/mapbox/flutter/FlutterMapboxPlugin.java

```diff
@@ -150,13 +150,11 @@ public void onMethodCall(MethodCall call, Result result) {
     case "easeTo": {
       long textureId = textureIdOfCall(call);
       if (maps.containsKey(textureId)) {
-
         CameraPosition cameraPosition = parseCamera(call.argument("camera"));
         int duration = intParamOfCall(call, "duration");
-        boolean easingInterpolator = booleanParamOfCall(call, "easingInterpolator");
 
         MapInstance mapHolder = maps.get(textureId);
-        mapHolder.map.easeTo(cameraPosition, duration, easingInterpolator);
+        mapHolder.map.easeTo(cameraPosition, duration, true);
```

Comment:
removing the boolean easing from the api, is a remainder from a legacy api that needs to be removed.